ykrt/src/mt.rs

```diff
@@ -431,7 +438,7 @@ impl MT {
     #[allow(clippy::not_unsafe_ptr_arg_deref)]
     pub fn control_point(self: &Arc<Self>, loc: &Location, frameaddr: *mut c_void, smid: u64) {
         match self.transition_control_point(loc, frameaddr) {
-            TransitionControlPoint::NoAction => (),
+            TransitionControlPoint::NoAction => {}
```

Comment:
?

Reply:
My local version of `cargo fmt` isn't changing this, and I'd be kinda surprised if it did. I wonder if you changed this manually, by accident, and then ascribed it to `cargo fmt`?

Reply:
Interesting, I tested it yesterday and it did remove, will check again.